### PR TITLE
WFLY-18495 microprofile-health Quickstart Common Enhancements CY2023Q3

### PR DESCRIPTION
**Pull request by kabir**
Supersedes #728

https://issues.redhat.com/browse/WFLY-18495

### CODE REVIEW

File: .github/workflows/quickstart_microprofile-health_ci.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths:
+      - 'microprofile-health/**'
+      - '.github/workflows/quickstart_ci.yml'
+jobs:
+  call-quickstart_ci:
+    uses: ./.github/workflows/quickstart_ci.yml
+    with:
+      QUICKSTART_PATH: microprofile-health
+      MICROPROFILE: true
```
Comment (emmartins) [inline .github/workflows/quickstart_microprofile-health_ci.yml:14]: should be TEST_BOOTABLE_JAR: true

**Review by emmartins — CHANGES_REQUESTED** [review]
those little things :-)

File: .github/workflows/quickstart_microprofile-health_ci.yml (hunk 1 above)
Comment (kabir) [reply .github/workflows/quickstart_microprofile-health_ci.yml:14]: Oh I forgot to look at this!

**Pull request merged**